libs/libdaemon/src/BaseDaemon.cpp

```diff
@@ -1172,10 +1172,10 @@ void BaseDaemon::initialize(Application & self)
 void BaseDaemon::logVersion() const
 {
     auto * log = &Logger::root();
-    LOG_INFO(log, "Welcome to TiFlash");
+    LOG_WARNING(log, "Welcome to TiFlash");
     std::stringstream ss;
     TiFlashBuildInfo::outputDetail(ss);
-    LOG_INFO(log, "TiFlash build info: {}", ss.str());
+    LOG_WARNING(log, "TiFlash build info: {}", ss.str());
```

Comment:
why?

Reply:
So we can find when TiFlash restarts in tiflash_error.log only.

Reply:
I don't think turning this logging to warning level is a good idea. It is confusing.

Reply:
Then it will be hard for us to locate the exact timestamp TiFlash restarts, and error logs around them. Since INFO logs is not as accessible as WARNING/ERROR logs.

Reply:
maybe turning the following logging of unclean restart to warning is more reasonable

https://github.com/pingcap/tiflash/blob/32aa07f2b9ee5df173b257ff7ee4a04c7528103e/dbms/src/Server/StatusFile.cpp#L46-L56

Reply:
I will change it back

Reply:
fixed

Reply:
Still warning?

Reply:
yes, sorry, it's really fixed now